docs/plugins/substitute.rst

```diff
@@ -0,0 +1,23 @@
+Substitute Plugin
+==============
+
+The ``substitute`` plugin lets you easily substitute values in your templates and
+path formats. Specifically, it is intended to let you *canonicalize* names
+such as artists: for example, perhaps you want albums from The Jimi Hendrix
+Experience to be sorted into the same folder as solo Hendrix albums.
+
+This plugin is intented as a replacement for the ``rewrite`` plugin. Indeed, while
```

Comment:
Not sure why the word "indeed" is here. Could also be without it: "While the rewrite plugin modifies...., this plugin does not."

Reply:
Also a little further above: Always capitalize after colon `:`

Reply:
Fixed